libr/debug/debug.c

```diff
@@ -386,6 +386,7 @@ R_API RDebug *r_debug_new(int hard) {
 	dbg->q_regs = NULL;
 	dbg->call_frames = NULL;
 	dbg->main_arena_resolved = false;
+	dbg->glibc_version = 2.31; /* default version ubuntu 20 */
```

Comment:
i see this double/float solution a bit tricky and could be probelmatic on some situations. i see different options. and having an api to handle `r_semver_new (2, 31, 0)` whjich may result in a struct or packed values into ut32 dword. 

r_semver_gt() to compare < or > between them.

Reply:
I agree the usage of double to store the version seem hacky. Better to replace it with something else.

Reply:
We need the r_semver api

Reply:
but this must be done in a separate PR and this PR will be blocked until then. Who will proceed on implementing such API?

Reply:
maybe if it's needed only here for now we can just use something small and local. The minimal things for this PR, to avoid blocking @meowmeowxw from going on with this.

Reply:
Should I keep it double for this pull request or change to something else?

Reply:
@meowmeowxw just shift it to be int `231` for now maybe? The proper API can be introduced later and in a separate PR imho.

Reply:
yeah use some int or something, but not double/float .